Samples/IonicCapacitorAngularV2/ct-demo-ionic-angular-cap/src/app/home/home.page.ts

```
@@ -0,0 +1,266 @@
+          startdate: Date(),
+          enddate: Date(),
+        };
+        CleverTap.recordEventWithNameAndProps('boo', map);
+        break;
+      case "Record Charged Event":
+        const details = {
+          amount: 300,
+          'Charged ID': 1234,
+          paymentdate: Date(),
+          transactiondate: Date(),
+        };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)